jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletApiResponse.java

```diff
@@ -559,11 +560,12 @@ public void reset()
 
         _response.reset();
 
+
         ServletApiRequest servletApiRequest = _response.getServletContextRequest().getServletApiRequest();
-        Session session = servletApiRequest.getCoreSession();
+        ManagedSession session = servletApiRequest.getServletContextRequest().getManagedSession();
```

Comment:
It is probably possible. But currently ManagedSession contains servlet specific features like passivation and listeners. It would need to be split to move those boys into some servlet common module.... That we don't yet have.  So a good thing to look at, but that's a different PR i think.